content/en/altinity-kb-integrations/altinity-kb-kafka/altinity-kb-kafka-grants.md

```diff
@@ -0,0 +1,66 @@
+# Grants and the Kafka Engine   
+
+There are some issues that are useful to know about when working with the Kafka engine with respect to Grants
+
+## Default user, out of the box
+
+The default user, as set up on a stock installation, has no issues with respect to access to both Kafka Engine tables or `system.kakfa_consumers`.
+
+## Creating users with access to Kafka engine tables and `system.kafka_consumers`
+
+There is sometimes a need to create a database user for the purpose of accessing various system tables, albeit one without admin privileges and one that is read-only.
+
+In this situation, the grants given aren't what one would expect in that grants would be given on a table-granular level, but at a database-wide level.
+
+The request: there needs to be a "dba" user with read-only privileges that has the ability to read `system.kafka_consumers`.
+
+Granting `SELECT` to the user specifying `system.kafka_consumers` and any Kafka engine tables, per table, will allow access to those tables and `system.kafka_cosumers` without any errors, but the resultset of selecting the rows in `system.kafka_consumers` will be empty.
+
+The only way for this to work is the following:
+
+```
+CREATE ROLE 'dbsupport';
+GRANT SELECT on *.* to dbsupport_admin;
```

Comment:
dbsupport ?

Reply:
Isn't SHOW TABLES enough?

Reply:
it is

Reply:
I'd be glad to take on the task of how to set up permissions for the system database in general. @SaltTan I can talk to you about it this week?